RPI/README.md

```diff
@@ -43,7 +45,7 @@ If you are cross-compiling, this would apply only to the compile host:
 ```
 ./mk/os-pkg/ubuntu-packages.sh
 sudo pip install -r  mk/python/pip_required_build.txt
-sudo pip install -r  mk/python/pip_required_gui.txt
+sudo pip install -r  mk/python/pip_required_gse.txt
```

Comment:
These are fine for now, but eventyally will need to be changed to (or the sudo equivalent):
```
pip install -r Autocoders/Python/requirements.txt
pip install -r Gds/requirements.txt
```

Reply:
I'll try this on a clean pi install, likely should just be updated to what you have :)